package.json

```diff
@@ -9,6 +9,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.6",
```

Comment:
Are these new dependencies used anywhere?

Reply:
Sorry, I've added this but never be used and forgot to delete it.